File: ubb/ubb_frontend/my-app/src/dto/BugTicketDto.ts
import { TinyProjectDto } from "./TinyProjectDto";
import { TinyUserDto } from "./TinyUserDto";

export interface BugTicketDto{

    id:number;
    project:TinyProjectDto;
    title:string;
    description:string;
    priority:string;
    status:string;
    createdBy:TinyUserDto;
    assignedTo:TinyUserDto;
    commentsIds: number[];

}

File: ubb/ubb/src/main/java/ubb/project/ubb/dto/BugTicketDto.java
package ubb.project.ubb.dto;

import lombok.Data;
import java.util.List;

@Data
public class BugTicketDto {


    private Long id;


    private TinyProjectDto project;

    private String title;

    private String description;

    private String priority;

    private String status;

    private TinyUserDto createdBy;

    private TinyUserDto assignedTo;

    private List<Long> commentsIds;
}
